IBX-6417: Describe UI/UX improvements (#264)
ibexa/documentation-user: docs/getting_started/discover_ui.md

# User interface

## Top bar

At the top of many screens you might see a top bar.
Depending on your location within the Back Office, it can contain the following important features:

- Logo in the left corner, which is a direct link to the [dashboard](#my-dashboard)

> - "Focus mode" callout which informs you that the [focus mode](#focus-mode) is on

Author: ```suggestion
- "Focus mode" badge which informs you that the [focus mode](#focus-mode) is on
```

The left side menu allows you to move between important sections of the application.

> ![Side menu](img/side_menu.png "Side menu")

Reviewer: https://github.com/ibexa/documentation-user/pull/246#discussion_r1467815076

> ![Side menu](img/side_menu.png "Side menu")

Reviewer: Please update the screenshot.

Depending on the product edition and your [permissions](../permission_management/permission_system.md), the top level sections on the leftmost pane may include, for example:

- **Content**, which gives you access to the content Repository.
It lets you navigate the Content Tree, create, edit, move, copy, delete content, etc.

> - [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]] **Site management**, which enables you to create and edit block-based Pages and manage multiple websites.

Reviewer: What does it mean block-based pages?

Author: I have no idea, not my wording. I guess the author meant Pages with blocks

!!! note

    If you have administrator permissions, you can also view and manage User accounts and site skeletons in the **Admin** tab.

    For more information about users and permissions, see [Users](../permission_management/permissions_and_users.md).

    For more information about site skeletons, see [Site skeletons]([[= developer_doc =]]/multisite/site_factory/site_factory_configuration/#site-skeletons).

## Focus mode

> Focus mode helps editors focus on information that is only relevant to their line of work and omit technical details that would otherwise introduce distraction.

Reviewer: ```suggestion
Focus mode helps editors focus on information that is only relevant to their line of work and omit technical details that would distract.
```

> It is enabled when you first log into [[= product_name =]], and influences the way information is displayed in the user interface.

Reviewer: ```suggestion
It is enabled by default, after you log in.
```

> If you need access to certain settings or technical details, you can [disable the focus mode](#disable-focus-mode) at any time.

Reviewer 2: ```suggestion
If you need am access to certain settings or technical details, you can [disable the focus mode](#disable-focus-mode) at any time.
```

> If you need access to certain settings or technical details, you can [disable the focus mode](#disable-focus-mode) at any time.

Reviewer: Once you use disable focus mode, once turn off. I think it should be consistent. Maybe change all entries with turn off to disable focus mode.

Dark Content Tree
: In areas where the Content Tree is displayed, for example, in **Content structure**, the tree is displayed on a dark background.

![Dark Content Tree](img/FM_dark_content_tree.png "Dark Content Tree")

Settings unavailable in Content area

> : Compared to the regular mode, in Focus mode, in **Content** area of the left-hand menu, the **Settings** part of the menu is not available.

Reviewer 2: ```suggestion
: Compared to the regular mode, in Focus mode, in **Content** tab, the **Settings** part of the menu is not available.
```

> ![Standard mode vs. Focus mode](img/FM_menu_settings.png "Content menu in standard mode vs. Focus mode")

Reviewer 2: Screenshot is blurred. Please update with better quality.

To display the Content item details view with more actions, click **Exit full view**.

![Content item in full view](img/FM_content_item_full_view.png "Content item in full view")

!!! tip

    Even when you're out of the full view or not in Focus mode, you can still preview the Content item in the **View** tab.

Different details view tabs in Focus mode and when it is disabled
: In Focus mode, the tabs in Content item's detail view are different than the ones visible when it is disabled.
Additionally, they are displayed in different order to expose the ones that are more important from the editor's perspective.

![Tabs in Content item's details view](img/FM_content_item_tabs_comparison.png "Content item tabs in standard mode vs. Focus mode")

Different columns available for selection
: You can decide which columns are displayed in the **Sub-items** table, to adjust the view to your use case.
The list of columns available for selection differs between the Focus mode and when it is disabled.

> ![Columns in Sub-items table](img/FM_subitems_column_list_comparison.png "Sub-items table columns in standard mode vs. Focus mode")

Reviewer 2: Screenshot is still blurred. Please update with better quality.
